searcher.go

```diff
@@ -18,7 +18,7 @@ import (
 const (
 	// DefaultWatchTimeout is the time to wait on watches against the Kubernetes
 	// API before giving up and throwing an error.
-	DefaultWatchTimeout = 90 * time.Second
+	DefaultWatchTimeout = 3 * time.Second
```

Comment:
My motivation to change this is that the resource handlers should not block for too long. Especially not by default. Whoever wants more timeout can configure it.